tests/core/snapd-failover/task.yaml

```diff
@@ -12,11 +12,18 @@ debug: |
     journalctl -u snapd.socket || true
     ls -l /snap/snapd/
 
+prepare: |
+    cp -a /etc/systemd/system/snapd.service.d/local.conf local.conf.bak
+    sed -i 's/^StartLimitInterval=0/StartLimitInterval=10s/g' /etc/systemd/system/snapd.service.d/local.conf
```

Comment:
Should we just generate the "correct" data?

Reply:
This is the correct data - or what do you mean with that?

Reply:
If this is the correct data, can we just have the local.conf file generated with it. Why should this be unique to the failover test?

Reply:
I'm happy to explore this further once we unblocked master. But my understanding is that the correct value is different for different tests. The change in https://github.com/snapcore/snapd/pull/8155/files#diff-556bb7431481e375713ea3e0883a771aR88 tries to address this but AIUI there are complications. I think we should not wait for perfect and just unblock master.

Reply:
it's very unfortunate that we need StartLimitInterval=0 for the tests generally, is not a very realistic setting. Anyway I agree we should unblock master